### PR TITLE
Make instructions to open file in editor clearer

### PR DESCRIPTION
**Pull request by jjant**
## Description

It took me a while to understand how to actually set up a binding to open the selected file from the linked issue in the readme, so I thought it'd be clearer to show an example with the actual code that will do it.

### CODE REVIEW

**Comment by PatrickF1** [comment]
Hi @jjant, thanks for opening up this PR! The readme is more for documentation and the cookbook is more for implementing customizations. Do you mind if I copy your example/code into the Cookbook instead?

**Comment by PatrickF1** [comment]
Ah I did have it in the Wiki but in the wrong section. My bad! https://github.com/PatrickF1/fzf.fish/wiki/Troubleshooting#custom-fzf-bindings-to-open-file-in-another-program-not-working. I'll also put it in the Cookbook.

File: README.md [hunk 1]
```diff
@@ -130,9 +130,19 @@ The following variables can store custom options that will be passed to fzf by t
...
 - add [key bindings](https://www.mankier.com/1/fzf#Key/Event_Bindings) within fzf to operate on the selected line:
-  - [open file in Vim](https://github.com/junegunn/fzf/issues/1360)
+
+  - [open file in in your `$EDITOR`](https://github.com/junegunn/fzf/issues/1360):
+
+    ```fish
+      # set -gx $EDITOR "nvim" # or "vim", or "code", etc.
+
+      # Ctrl-o will open the selected file/directory in your editor of choice.
+      set fzf_dir_opts --bind "ctrl-o:execute($EDITOR {} &> /dev/tty)"
```
Comment (PatrickF1) [inline README.md:140]: in case anyone is referencing this in the future, `fzf_dir_opts` has been renamed to `fzf_directory_opts`.

**Comment by jjant** [comment]
@PatrickF1 Thanks for taking a look at the PR!
 I hadn't seen the cookbook (tho admittedly, GH wikis are not that discoverable 😅).

**Pull request closed without merging**